share/commands/agent-inventory

```diff
@@ -14,26 +14,11 @@
 # @man *-d*: run the agent in debug mode, prints low-level information
 # @man +
 # @man *-c*: run the agent without color output
+# @mab +
+# @man *-r*: run the agent with raw output
+# @mab +
+# @man *-R*: run the agent in completely unparsed mode, with no return code of 1 in case of error. A little faster.
+# @mab +
+# @man *-m*: run the agent with multiline output
```

Comment:
Same as agent-run, but without class and bundle selection (to be sure we are really doing an inventory).